app/src/main/java/com/example/playlistmaker/SettingsActivity.kt

```diff
@@ -15,10 +17,35 @@ class SettingsActivity : AppCompatActivity() {
         setContentView(R.layout.activity_settings)
 
         val buttonBack = findViewById<Toolbar>(R.id.settings_back)
-
         buttonBack.setOnClickListener {
             finish()
         }
 
+        val buttonShare = findViewById<TextView>(R.id.share)
+        buttonShare.setOnClickListener {
+            val message = getString(R.string.link)
+            val intent = Intent(android.content.Intent.ACTION_SEND);
+            intent.setType("text/plain");
+            intent.putExtra(android.content.Intent.EXTRA_TEXT, message);
```

Comment:
Путь до класса тут лишний, так как есть импорт:
```suggestion
            val intent = Intent(Intent.ACTION_SEND);
            intent.setType("text/plain");
            intent.putExtra(Intent.EXTRA_TEXT, message);
```